TypeCobol/Compiler/Domain/ProgramSymbolTableBuilder.cs

```diff
@@ -246,6 +246,74 @@ public override void EndLinkageSection()
             CurrentDataDivisionSection = null;
         }
 
+        /// <summary>
+        /// Resolve in the current program linkage section, the given parameter.
+        /// </summary>
+        /// <param name="p">The parameter to resolve.</param>
+        /// <returns>The resolved variable if any, null otherwise</returns>
+        private VariableSymbol ResolveUsingParameter(CallTargetParameter p)
```

Comment:
This code has been removed from develop branch. It must be removed as weel here.

Reply:
Done